test/runnable/xtest46.d

```diff
@@ -7866,7 +7867,7 @@ bool test16022()
 {
```

Comment:
This [issue 16022](https://issues.dlang.org/show_bug.cgi?id=16022) was very specific to the comma operator.  Since the result of a comma expression now emits an error, this test should no longer compile, and should probably be moved to the fail_compilation test suite.

Reply:
Ok. I copied it to fail compilation. I don't see any reason to remove it though.